poppy/poppy_core.py

```diff
@@ -2738,10 +2740,10 @@ def _find_pixelscale_in_headers(keywords, headers):
             if pixelscale is None and self.planetype is None:
                 # we don't know which keywords might be present yet, so check for both keywords
                 # in both header objects (at least one must be non-None at this point!)
-                _log.debug("  Looking for 'PUPLSCAL' or 'PIXSCALE' in FITS headers to set "
+                _log.debug("  Looking for 'PUPLSCAL' or 'PIXSCALE' or 'PIXELSCL' in FITS headers to set "
                            "pixel scale")
                 keyword, self.pixelscale = _find_pixelscale_in_headers(
-                    ('PUPLSCAL', 'PIXSCALE'),
+                    ('PUPLSCAL', 'PIXSCALE','PIXELSCL'),
```

Comment:
Wait never mind I read that backwards, you did this right and I confused before and after in the diff. 

Too many hours in a row moving mirrors at Goddard, I'm not thinking straight.